portal-kernel/src/com/liferay/portal/kernel/security/permission/ResourceActionsUtil.java

```diff
@@ -277,6 +277,24 @@ public static boolean isRootModelResource(String modelResource) {
 		return getResourceActions().isRootModelResource(modelResource);
 	}
 
+	public static void read(
```

Comment:
@dantewang @yuhai Why we need this version of API? You have already the one with multiple arguments.


Tina.

Reply:
@tinatian, I will remove the api.